### ISSUE TITLE
[Feature Request] More space on Favorites list for scrolling

### ISSUE THREAD

**Issue by jcperil** [falzonv/discreet-launcher#271]
Hi,

I don't know if this will be an issue for all but I have trouble scrolling the favorites list. Probably due to my fingers, I often mistakenly touch the reordering up/down arrows and end up rearranging some apps. Also, sidenote, the scrolling area is sometimes insensitive to touch.

<img src="https://user-images.githubusercontent.com/89805719/216935506-6f6fac50-4e4d-40be-9d92-05f71f373d07.jpg" height="300px"/>

I feel like there should be more space to scroll. I have come up with three ideas. Please let me know if any of these are feasible to implement.

---
1. Provide more space to scroll on both sides. Although, this would mean that apps with very long names might not be shown completely.

<img src="https://user-images.githubusercontent.com/89805719/216940946-16fb68be-710a-42b7-90cb-284fa170e89b.png" height="300px"/>

---
2. Move the list to left side, let's say. Then there would be enough space on the right side to scroll.

<img src="https://user-images.githubusercontent.com/89805719/216941069-2caed8d2-7c1a-4864-af01-8a659ee46754.jpg" height="300px"/>

---
3. Allow only scrolling and disable any reordering of apps by default. Activate reordering only if you long press an app.

**Comment by falzonv** [comment]
Hello,

Thank you for these very detailed and clear suggestions :-)

After trying them, I decided to go with **option 2** because option 1 doesn't really look good on small screens and option 3 would involve too much internal changes.
I also found how to display a scrollbar to the right of the scrolling view so these changes (aligning to the left, adding a scrollbar) should make the interface much better in the next release.

> Also, sidenote, the scrolling area is sometimes insensitive to touch.

=> Strange, I never encountered this behavior but maybe it is related to the area being too small, let's see if you still observe this issue after the changes described above are implemented.

Best regards.

**Comment by jcperil** [comment]
Thank you. Waiting for the update :)

**Comment by falzonv** [comment]
Hello,

I am pleased to inform you that these changes have been implemented in new release v7.2.0!
This release has been published few minutes ago.

Here are the usual availability delays for each location:
- 4 to 8 days on F-Droid
- 0 to 2 days on Google Play
- Already available as direct APK on GitHub (https://github.com/falzonv/discreet-launcher/releases)

*Remarks:
I will leave this ticket opened until the release is effectively published on F-Droid.
You can use the export/import feature to switch from one version to another.*

Best regards.

**Comment by falzonv** [comment]
Hello,

> Congratulations on the second anniversary. :)

Thank you!

> > You can use the export/import feature to switch from one version to another.
>
> Do you have plans for [reproducible builds](https://f-droid.org/docs/Reproducible_Builds/)?

Not really... Looking at the large section "Potential sources of unreproducible builds" of this link, it seems there are many things which can go wrong in the process, or need to be adjusted before everything works as expected. And it also seems to require several additional steps when publishing a new version, a process that I already found quite tedious(\*).

I don't have much time to work on the launcher now, compared to when I started in 2021, so I prefer to focus on improving the launcher features or fixing potential bugs.

*(\*) For information, the current steps when publishing a new version are to: make sure everything is ready (tests), write the release notes and change the version in the code, create the git tag for F-Droid, build the APK for GitHub, build the AAB for Google Play, create a release in GitHub with its notes and upload the APK as attached file, connect to Google Play console and add a new release, upload the AAB and write the Google Play release notes. Also if the metadata need to be adjusted, in Google Play I have to do it manually for each translation (currently 12).
Actually F-Droid has the simplest process right now, so I would prefer to keep it that way ^^*

Best regards.
